content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Istio Vault CA Integration
+description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+This tutorial walks you through an example to plug in a Vault CA for issuing
+certificates in Istio.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
+
+## Install Istio with SDS enabled
+
+1.  Install Istio with SDS enabled using [Helm](/docs/setup/kubernetes/helm-install/#prerequisites)
+and Node Agent sending certificate signing
+requests to a testing Vault CA:
+
+    {{< text bash >}}
+    $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
+    $ helm dep update --skip-refresh install/kubernetes/helm/istio
+    $ cat install/kubernetes/namespace.yaml > istio-auth.yaml
+    $ cat install/kubernetes/helm/istio-init/files/crd-* >> istio-auth.yaml
+    $ helm template \
+        --name=istio \
+        --namespace=istio-system \
+        --set global.mtls.enabled=true \
+        --set global.controlPlaneSecurityEnabled=true \
+        --set global.proxy.excludeIPRanges="35.233.249.249/32" \
+        --values install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml \
+        install/kubernetes/helm/istio >> istio-auth.yaml
+    $ kubectl create -f istio-auth.yaml
+    {{< /text >}}
+
+The testing Vault server used in this tutorial has the IP
+address `35.233.249.249`. The configuration
+`global.proxy.excludeIPRanges="35.233.249.249/32"` whitelists the IP address of
+the testing Vault server, so that Envoy will not intercept the traffic from
+Node Agent to Vault.
+
+The yaml file [`values-istio-example-sds-vault.yaml`]({{< github_file >}}install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml)
+contains the configuration that enables SDS (secret discovery service) in Istio.
+The Vault CA related configuration is set as environmental variables:
+
+{{< text yaml >}}
+env:
+- name: CA_ADDR
+  value: "https://35.233.249.249:8200"
+- name: CA_PROVIDER
+  value: "VaultCA"
+- name: "VAULT_ADDR"
+  value: "https://35.233.249.249:8200"
+- name: "VAULT_AUTH_PATH"
+  value: "auth/kubernetes/login"
+- name: "VAULT_ROLE"
+  value: "istio-cert"
+- name: "VAULT_SIGN_CSR_PATH"
+  value: "istio_ca/sign/istio-pki-role"
+{{< /text >}}
+
+## Deploy a testing workload
+
+This section deploys a testing workload `httpbin`. When the sidecar of the
+testing workload requests a certificate through SDS, Node Agent will send
+certificate signing requests to Vault.
+
+1.  Generate the deployment for an example `httpbin` backend:
+
+    {{< text bash >}}
+    $ istioctl kube-inject -f @@samples/httpbin/httpbin.yaml@@ > httpbin-injected.yaml
```

Comment:
Sorry, my comment was ambiguous. Just a single @ on either end is what is needed, not two.

Reply:
Done.